boltbrowser.go

```diff
@@ -47,8 +56,12 @@ func main() {
 	databaseFiles := flag.Args()
 	for _, databaseFile := range databaseFiles {
 		currentFilename = databaseFile
-		db, err = bolt.Open(databaseFile, 0600, nil)
-		if err != nil {
+		db, err = bolt.Open(databaseFile, 0600, &bolt.Options{Timeout: DefaultDBOpenTimeout})
```

Comment:
It looks like you're always passing the Default timeout to bolt. Update this to DBOpenTimeout instead of DefaultDBOpenTimeout and I think we're good!

Reply:
Good catch 👍

Reply:
Fixed